docs/fundamentals/networking/telemetry/metrics.md

```diff
@@ -17,16 +17,20 @@ They are also [multi-dimensional](../../../core/diagnostics/metrics-instrumentat
 > [!TIP]
 > For a comprehensive list of all built-in instruments together with their attributes, see [System.Net metrics](../../../core/diagnostics/built-in-metrics-system-net.md).
 
-## Collect System.Net metrics
+## Collecting System.Net metrics
 
-There are two parts to using metrics in a .NET app:
+In order to take advangage of the built-in metrics instrumentation, a .NET app needs to be configured to collect these metrics. This typically means transforming them for external storage and analysis, e.g., to monitoring systems.
```

Comment:
Did you edit these files in VS Code? If so, it would be good to run Acrolinx on them to find a lot of the stuff that I would otherwise find. See https://review.learn.microsoft.com/en-us/help/platform/acrolinx-vscode-setup?branch=main#install-acrolinx-in-visual-studio-code.